dix/req.c

```diff
@@ -2323,7 +2325,7 @@ M0_INTERNAL int m0_dix_put(struct m0_dix_req      *req,
 	M0_PRE(keys->ov_vec.v_nr == vals->ov_vec.v_nr);
 	M0_PRE(keys_nr != 0);
 	/* Only overwrite, crow and sync_wait flags are allowed. */
-	M0_PRE((flags & ~(COF_OVERWRITE | COF_CROW | COF_SYNC_WAIT)) == 0);
+	M0_PRE((flags & ~(COF_OVERWRITE | COF_CROW | COF_SYNC_WAIT | COF_SKIP_LAYOUT)) == 0);
```

Comment:
Use M0_IN() here and also update the comment above it.

Reply:
Done.